src/workspace-tree/bazel_workspace_tree_provider.ts

```diff
@@ -55,17 +63,36 @@ export class BazelWorkspaceTreeProvider
       false,
       false,
     );
+
+    // Initialize disposables array if it's not already initialized
```

Comment:
What's the point of initialising it like this, rather than like how it used to be?